src/Analyzers/CSharp/CodeFixes/RemoveUnnecessaryParentheses/CSharpRemoveUnnecessaryParenthesesCodeFixProvider.cs

```diff
@@ -24,9 +24,7 @@ protected override bool CanRemoveParentheses(SyntaxNode current, SemanticModel s
             => current switch
             {
                 ParenthesizedExpressionSyntax p => CSharpRemoveUnnecessaryExpressionParenthesesDiagnosticAnalyzer.CanRemoveParenthesesHelper(p, semanticModel, out _, out _),
-#if !CODE_STYLE
```

Comment:
do how do you find all these cases?  manual auditing of CODE_STYLE?  Should we have a different pattern we use to mark CODE_STYLE sections against a particular version so it's easier to find/delete?

Reply:
It was manual, but not too difficult. I don't think we need to make any changes at this time.